client/utils/types.ts

```diff
@@ -0,0 +1,11 @@
+export type FromValue = HTMLInputElement[ 'value' ];
+
+export type FormInputProps = {
```

Comment:
A todo comment here explaining that this can be moved into the `Form` component once it's converted to TS would be really helpful 😄

Reply:
Added in 0f18e7f91ea08b31812e9a7d3ad8518244ae509b. Thanks!